thirdeye/thirdeye-frontend/app/pods/rootcause/template.hbs

```diff
@@ -59,4 +59,9 @@ urns
   columns=eventTableColumns
   selectedUrns=selectedUrns
   onSelect=(action "tableOnSelect")
+}}
+
+<h3>This is the heatmap.</h3>
+{{rootcause-heatmap
+  mode='change'
```

Comment:
fyi, we `generally` follow this convention:  use single quotes in JS, double quotes in the template.

Reply:
fixed